org.eclipse.january.examples/src/org/eclipse/january/examples/dataset/SlicingExamples.java

```diff
@@ -41,6 +43,90 @@
 	public void before() {
 		Utils.suppressSLF4JError();
 	}
+	
+	/**
+	 * Slice using basic 1D int array. The goal is to take the data from index 1
+	 * @throws Exception 
+	 */
+	@Test
+	public void sliceFrom1DFromIndex1() throws DatasetException {
+		final Dataset onedData = DatasetFactory.createFromObject(new int[]{1,2,3});
+		Dataset sliceData = onedData.getSlice(new Slice(1, null, null));
+		System.out.println(sliceData.toString(true));
+	}
+	
+	/**
+	 * Slice using basic 1D int array. The goal is to create a Slice with only the stop index
+	 * @throws Exception 
+	 */
+	@Test
+	public void sliceFrom1DWithStopIndex() throws DatasetException {
+		final Dataset onedData = DatasetFactory.createFromObject(new int[]{1,2,3});
+		Dataset sliceData = onedData.getSlice(new Slice(2));
+		System.out.println(sliceData.toString(true));
+	}
+	
+	
+	/**
+	 * Slice using basic 1D int array. The goal is to get the middle of the array
+	 * @throws Exception 
+	 */
+	@Test
+	public void sliceFrom1D() throws DatasetException {
+		final Dataset onedData = DatasetFactory.createFromObject(new int[]{10,9,8,7,6,5,4,3,2,1,0});
+		//we want to get the 3 values middle of the array, that's to say 6,5,4
+		//This mean that you get the value at the 4th position to the value at the 6th position because the second one is exclusive, so you get at the index of end-1
```

Comment:
This language is confusing between "position" and "index". IIRC position is normally 1-based, but index is 0-based.